browser/BUILD.gn

```diff
@@ -50,6 +50,9 @@ group("browser") {
 
   # Group depending on upstream's counterpart to make it easier to manage
   # dependencies in brave/ (//chrome/browser does not depend on this target
-  # so //brave/browser becomes essentially an alias for //chrome/browser now).
-  public_deps = [ "//chrome/browser" ]
+  # so //brave/browser acts as a an alias for //chrome/browser now).
+  public_deps = [
+    ":browser_process",
```

Comment:
why did this need to be added here?

Reply:
I'm removing a duplicated reference to the brave_browser_process.h header file from //chrome/browser as it was also listed in //brave/browser:browser_process, so added this in order to make sure that anything in Brave including //brave/browser would get this as well, as it would before via the other public_dep.

But perhaps you'd prefer perhaps to leave things as they were, that would be ok for me too.

Reply:
@bridiver Fixed